### PR TITLE
Frontend UI stock discovery category name updates

### PR DESCRIPTION
**Pull request by BearachB**
# Changes
1. Added all sectors to Discovery page (each sector displays the top 20 positive movers over the past 24h period).
2. Added a hierarchy of titles (i.e. Top movers -> Top Gainers | Top Losers. Top Stocks By Sector -> Top Industrials | Top Technology)
3. Renamed almost all of the categories to snappier names.
4. Added icons to each category using MaterialUI.
5. Added info buttons to most categories.
6. Added a grey bottom border between each major section.

There is probably going to need to be a bit of discussion on this as we go as what everyone likes is going to be pretty subjective. There's some aspects of this page that I'm still undecided on, however I think it's visually better than the last iteration.

# Screenshots
## Recommended section
image
## Top ESG
image
## Info Button
image
## Footer
image

## Screenshot of React Console
image

### CODE REVIEW

File: src/components/stockDiscoveryComponents/stockSummary/stockSummary.js [hunk 1]
```diff
@@ -6,7 +6,29 @@ import MessageAlert from '../../widgets/MessageAlert/MessageAlert';
...
+import LanIcon from '@mui/icons-material/Lan';
+import AttachMoneyIcon from '@mui/icons-material/AttachMoney';
+import LocalHospitalIcon from '@mui/icons-material/LocalHospital';
+import PeopleIcon from '@mui/icons-material/People';
+import GroupsIcon from '@mui/icons-material/Groups';
+import HomeIcon from '@mui/icons-material/Home';
+import ElectricalServicesIcon from '@mui/icons-material/ElectricalServices';
+import CloudIcon from '@mui/icons-material/Cloud';
+import ScienceIcon from '@mui/icons-material/Science';
+import ElectricBoltIcon from '@mui/icons-material/ElectricBolt';
```
Comment (WarrenK-design) [inline src/components/stockDiscoveryComponents/stockSummary/stockSummary.js:31]: For these not massive but you can destructure the imports and get away with these in one line. 

```
import { IconName1, IconName2, IconName3, IconeName4 } from '@mui/icons-material';
```

**Review by WarrenK-design — DISMISSED** [review]
Hi Bearach, Good work just a few things. 

# Footer 
When the page is loading it looks really odd because the footer moves all the way up the page.
I'd be in favour of just removing  it all together at this stage not worth spending time on to fix.
image

# Line breaks 
I think these two clash, took me a min to get it. 
image

When I go to mobile then I taught it was actually the scroll bar confused me a bit.  
image

# Explanations 
This only a small one but first line of these is bit repetitive and I know there's title but takes bitta work to see what they actually ranked on, a fix could just be to bold/colour what is being measured in the text maybe? Green -> Positive , Red -> Negative and bold for others maybe? 
image
image
image
image
image
image

**Comment by joeycor92** [comment]
Looking good. First thing I noticed on desktop though is that it's not clear that there's more down further.
image

Maybe reduce the spacing. There's a lot of whitespace there on the page. 

Agreed about the footer, noticed that when the page was loading.

Just on the wording, I can see what Warren means about the repetition, it keeps saying "This section shows the top 20...". Just wondering if there's a way around it. Do we need to say top 20?

File: src/components/stockDiscoveryComponents/stockSummary/stockSummary.js (hunk 1 above)
Comment (BearachB) [reply src/components/stockDiscoveryComponents/stockSummary/stockSummary.js:31]: Good spot, addressed this locally, it'll be in the next push.

**Comment by beckskelly** [comment]
Hey Bearach, looking good, I'd get rid of those horizontal lines and remove the spacing like the others have said, but I like this overall!

**Comment by WarrenK-design** [comment]
> Looking good. First thing I noticed on desktop though is that it's not clear that there's more down further. image
> 
> Maybe reduce the spacing. There's a lot of whitespace there on the page.

I think adding scroll bar on website would fix this just to show it's scrollable, people who maybe cant use mouse wheel and only click would be able to use website too.

**Comment by BearachB** [comment]
Thanks for the comments @WarrenK-design, @joeycor92, @beckskelly!

I think I've addressed all the above comments in this latest push, let me know if there's anything else you want me to cover:
- Footer now loads in with the stock summary so it isn't hanging there on its own while the rest is loading.
- Section borders have been removed.
- Blue background added to the section titles.
- Text in the info buttons has been coloured for positive/negative and bolded in other places.
- @beckskelly said she would add in the scrollbar in her branch she's on now. It would take too long to explain how to do it to me so she said it would be easier overall to do it on hers.

# Screenshots
## No Footer Until Load
image

## Blue Background (Also with white info buttons, and no more section borders)
image

## Info Button Coloured Text:
image
image
image

**Review by joeycor92 — COMMENTED** [review]
Lot of serious work done here. Can definitely see the benefit of all the changes and really like the updates to the information modals. 

The one thing for me is that the banners and titles all seem very big on desktop and they are taking up a lot of space on the screen. Should we consider making the titles smaller and decreasing the margins/padding? 

Happy to approve if everyone else is happy with it.

**Review by joeycor92 — APPROVED** [review]

**Comment by WarrenK-design** [comment]
Hi Bearach, 

It looks good but as discussed feel its a bit busy and the search bar gets alittle lost within it. 
Sorta looks like each section is a mini footer along the page. 
image
Dont mind if change to a plainer solution like below or @beckskelly had a solution maybe some underline if we really wanted?
image
For these then too just small thing, think we can say top once in section heading then remove it from each of these? 
image

**Comment by BearachB** [comment]
OK @WarrenK-design addressed your two changes in the latest push.
- Removed "Top" from sector categories.
- Removed blue banner from behind section headings.  

Still not fully sold on the lack of colour, we really could do with some form of background images to break it up however I don't exactly know how to go about that. Still looking very serious.

**Review by beckskelly — APPROVED** [review]
Happy with it for now, I'll have a think over the weekend about making the page a bit more fun, its low on fun and colour for now

**Review by WarrenK-design — APPROVED** [review]

**Pull request merged**